src/Model/Page.php

```diff
@@ -306,7 +306,7 @@ public function getName()
      */
     public function setSlug($slug)
     {
-        $this->slug = self::slugify(trim($slug));
+        $this->slug = self::slugify(trim((string)$slug));
```

Comment:
missing space between the cast and the variable, this is why the build fails. Please amend your commit.

Reply:
this is what you suggested
```
so the fix would be https://github.com/sonata-project/SonataPageBundle/blob/3.x/src/Model/Page.php#L309, adding (string) before $slug
```
that's simply what was done above, no?

Reply:
@ob1201 you just need to add ` `  between `(string)` and `$slug` so you get `$this->slug = self::slugify(trim((string) $slug));`